integration_tests/src/main/python/hybrid_parquet_test.py

```diff
@@ -145,3 +145,67 @@ def test_hybrid_parquet_read_fallback_to_gpu(spark_tmp_path, parquet_gens):
             'spark.sql.sources.useV1SourceList': 'parquet',
             'spark.rapids.sql.parquet.useHybridReader': 'true',
         })
+
+@pytest.mark.skipif(is_databricks_runtime(), reason="Hybrid feature does not support Databricks currently")
+@pytest.mark.skipif(not is_hybrid_backend_loaded(), reason="HybridScan specialized tests")
+@hybrid_test
+def test_hybrid_parquet_filter_pushdown_gpu(spark_tmp_path):
+    data_path = spark_tmp_path + '/PARQUET_DATA'
+    with_cpu_session(
+        lambda spark: gen_df(spark, [('a', StringGen(pattern='[0-9]{1,5}'))]).write.parquet(data_path),
+        conf=rebase_write_corrected_conf)
+    # filter conditions should remain on the GPU
```

Comment:
How to verify this?

Reply:
I assumed that the startsWith is not supported in CPU but actually it does but not in their doc. If we pushed a unsupported operator to CPU the test should failed. I will try to find an expr that hybrid is not supported in this test.

Do you think it is necessary to write some UT to verify the executed plan?

Reply:
It will be straightforward if checking execution plan using UT.
IMO, we may use UT instead of IT. @sperlingxx  what do you think?

Reply:
Updated the integration tests to verify the executed plan, PTAL